src/components/Lend/LoanSearch/ChallengeCallout.vue

```diff
@@ -123,8 +123,8 @@ export default {
 		},
 		headerCallout() {
 			return this.shareLenderName
-				? `Support ${this.shareLenderName} and help ${this.teamName} hit their goal`
-				: `Help ${this.teamName} hit their goal`;
+				? `Support ${this.shareLenderName} and help `
+				: 'Help ';
```

Comment:
Does this trailing space create a double space in the message since a space also exists in the HTML?